apps/web/src/app/admin-console/organizations/layouts/organization-layout.component.ts

```diff
@@ -140,6 +143,14 @@ export class OrganizationLayoutComponent implements OnInit {
     ))
       ? "claimedDomains"
       : "domainVerification";
+
+    this.canShowPoliciesTab$ = this.organization$.pipe(
+      switchMap((organization) =>
+        this.organizationBillingService
+          .isUpsellingPoliciesEnabled$(organization)
+          .pipe(map((isUpsellingEnabled) => isUpsellingEnabled || organization.canManagePolicies)),
```

Comment:
⛏️ best practice for rxjs is to avoid nesting pipes. You should be able to do:
```suggestion
switchMap((organization) => this.organizationBillingService.isUpsellingPoliciesEnabled$(organization)),
map((isUpsellingEnabled) => isUpsellingEnabled || organization.canManagePolicies)
```